packages/cardpay-sdk/sdk/scheduled-payment-module.ts

```diff
@@ -1289,13 +1290,26 @@ export default class ScheduledPaymentModule {
     });
     let gasStationResponse = await getCurrentGasPrice(chainId);
     let gas = BigNumber.from((await gasEstimationResponse.json()).data?.attributes?.gas);
+
+    let tokenList = getConstantByNetwork('tokenList', network as SchedulerCapableNetworks)?.tokens;
+    let tokenUSD = tokenList.find((t) => t.symbol === 'USDT');
```

Comment:
Which USD token can I use for the conversion?

Reply:
Is it important that the usd stablecoin is in our token list available for payments? Can we just hardcode it to USDT or USDC? The latter seems more regulated and thus a safer choice but they are both supposed to be pegged so it doesn't really matter, right?

Reply:
yeah I think USDC is probably a safer choice. well they are both stable coins, pegged to USD. but if the peg ever drops (like it did for Terra), then it will matter. so mostly it;s about choosing the token that seems less likely to loose its peg, which is becomes, which token is better collateralized and hence has less risk of becoming insolvent.